fastfield_codecs/src/column.rs

```diff
@@ -242,7 +258,7 @@ mod tests {
     fn test_monotonic_mapping() {
         let vals = &[1u64, 3u64][..];
         let col = VecColumn::from(vals);
-        let mapped = monotonic_map_column(col, |el| el + 4);
+        let mapped = monotonic_map_column(col, |el| el + 4, |el| el);
```

Comment:
I don't understand. Isn't the second mapping supposed to be the inverse mapping?

Reply:
it's unused I replaced it with `unimplemented()`